frontend/src/scenes/onboarding/OnboardingVerificationStep.tsx

```diff
@@ -31,13 +31,14 @@ export const OnboardingVerificationStep = ({
     return !currentTeam?.[teamPropertyToVerify] ? (
         <OnboardingStep
             title={`Listening for ${listeningForName}s...`}
-            subtitle={`Once you have integrated the snippet, we will verify the ${listeningForName} was properly received. It can take up to 2 minutes to recieve the ${listeningForName}.`}
+            subtitle={`We're verifying that you've integrated the snippet and are sending ${listeningForName}s to PostHog. It can take up to 2 minutes to recieve an ${listeningForName}.`}
```

Comment:
```suggestion
            subtitle={`We're verifying that you've integrated the snippet or SDK and are sending ${listeningForName}s to PostHog. It can take up to 2 minutes to receive ${listeningForName}s.`}
```

`an recording` isn't correct 😄